pkg/skaffold/render/generate/generate_test.go

```diff
@@ -136,31 +138,50 @@ func TestGenerate(t *testing.T) {
 			},
 			expected: manifest.ManifestList{[]byte(podYaml), []byte(podsYaml)},
 		},
-		/* disabled
 		{
 			description: "render kustomize manifests",
 			generateConfig: latest.Generate{
-				Kustomize: []string{"base"},
+				Kustomize: &latest.Kustomize{
+					Paths: []string{"base"},
+				},
 			},
-			commands: testutil.CmdRunOut("kustomize build base", kustomizePatchedOutput),
-			expected: manifest.ManifestList{[]byte(kustomizePatchedOutput)},
+			commands: testutil.CmdRunOut("kustomize build base", podsYaml),
+			expected: manifest.ManifestList{[]byte(podsYaml)},
 		},
 		{
-			description: "render kpt manifests",
+			description: "render kustomize manifests - kubectl",
 			generateConfig: latest.Generate{
-				Kpt: []string{filepath.Join("fn", "Kptfile")},
+				Kustomize: &latest.Kustomize{
+					Paths: []string{"base"},
+				},
 			},
-			// Using "filepath" to join path so as the result can fix when running in either linux or
-			// windows (skaffold integration test).
-			commands: testutil.CmdRun(fmt.Sprintf("kpt fn render fn --output=%v",
-				filepath.Join(".kpt-pipeline", "fn"))),
-			expected: manifest.ManifestList{},
+			useKubectlKustomize: true,
+			commands:            testutil.CmdRunOut("kustomize build base", patchYaml),
+			expected:            manifest.ManifestList{[]byte(patchYaml)},
 		},
+		/*
```

Comment:
nit: should this test be removed (vs commented)?

Reply:
fixed the test